modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -10,9 +10,9 @@
 NFS is not recommended to use with metering. 
 ====
 
-Metering has no storage by default, but it can use any `ReadWriteMany PersistentVolume` or any `StorageClass` that provisions a `ReadWriteMany PersistentVolume`.
+Metering has no storage by default, but it can use any ReadWriteMany PersistentVolume or any StorageClass that provisions a ReadWriteMany PersistentVolume.
 
-To use a `ReadWriteMany PersistentVolume` for storage, modify the `shared-storage.yaml` file below.
+To use a ReadWriteMany PersistentVolume for storage, modify the `shared-storage.yaml` file below.
```

Comment:
Add ".Procedure"

Reply:
The step in a single-step procedure needs to be presented as a unordered list item.